plinth/modules/ejabberd/views.py

```diff
@@ -33,8 +37,52 @@ class EjabberdServiceView(ServiceView):
     template_name = 'ejabberd.html'
     description = ejabberd.description
     diagnostics_module_name = 'ejabberd'
+    form_class = EjabberdForm
+
+    def get_initial(self):
+        initdict = super().get_initial()
+        initdict.update({'MAM_enabled': self.is_MAM_enabled()})
+        return initdict
 
     def get_context_data(self, *args, **kwargs):
+        """Add service to the context data."""
         context = super().get_context_data(*args, **kwargs)
-        context['domainname'] = ejabberd.get_domainname()
```

Comment:
Was this line removed by accident?

Also, for the comment "Add service to the context data", should that be changed?